lib/telegram/bot/types/chat_member.rb

```diff
@@ -3,8 +3,14 @@
 module Telegram
   module Bot
     module Types
-      class ChatMember < Base
-      end
+      CHAT_MEMBER = (
```

Comment:
What if instead `CHAT_MEMBER` we define `ChatMember` for union type? [dry-types](https://dry-rb.org/gems/dry-types/1.7/sum/) does the same thing with `Types::Bool`. @AlexWayfer WDYT?

Reply:
I've made this firstly, but I've got the RuboCop offense:

![image](https://github.com/atipugin/telegram-bot-ruby/assets/6510020/e09e15aa-f309-4354-8560-71c9b0f23f5a)

So… if you want so, for consistency for example, I'm not against — I'll change and add ignorance for RuboCop.

Reply:
Done.